ui/src/utils/patchTests.ts

```diff
@@ -0,0 +1,16 @@
+import { Test } from '../types';
+import { checkForErrors } from '../utils/checkForErrors';
+
+export const patchTestsBySlot = (slotId: string, tests: Test[]) => {
+  const apiPrefix = process.env.REACT_APP_AUTOMAT_API_URL;
+  const apiUrl = `${apiPrefix}/admin/slotss/${slotId}`;
+  return fetch(apiUrl, {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ tests: tests }),
+  })
+    .then(checkForErrors)
+    .then((response) => response.json());
+};
```

Comment:
I wonder if defining all operations against the API in a single client would make sense? I think there will be a bunch of common building blocks that all operations have in common. What do you think?

Reply:
Yeah, I think it does. No need for each of these functions to live in a separate file at least. I'll follow up with a separate PR to capture the variants/filter work as well.